lib/xcpretty/formatters/simple.rb

```diff
@@ -8,12 +8,14 @@ class Simple < Formatter
     PASS = "✓"
     FAIL = "✗"
     PENDING = "⧖"
+    MEASURE = "⧗"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.